appserver/appclient/client/acc/src/main/java/org/glassfish/appclient/client/acc/ACCClassLoader.java

```diff
@@ -18,31 +18,26 @@
 
 package org.glassfish.appclient.client.acc;
 
+import static java.security.AccessController.doPrivileged;
```

Comment:
`doPriveleged()` at lines 69 and 134 are OK?

Reply:
I can still remove them. But this PR just takes a first step at removing them. Many, many more need to be removed. My plan is to do that in followup PRs.